calamari_ocr/ocr/datasets/input_dataset.py

```diff
@@ -322,12 +323,12 @@ def to_raw_input_dataset(self, processes=1, progress_bar=False, text_only=False)
         preloaded_datas, preloaded_texts, preloaded_params = datas, texts, params
         self._generate_only_non_augmented.value = prev
 
-        if self.dataset.mode == DataSetMode.TRAIN and self.data_augmentation_amount > 0:
+        if (self.dataset.mode == DataSetMode.TRAIN or self.dataset.mode == DataSetMode.PRED_AND_EVAL) and self.data_augmentation_amount > 0:
             abs_n_augs = int(self.data_augmentation_amount) if self.data_augmentation_amount >= 1 else int(self.data_augmentation_amount * len(self))
             preloaded_datas, preloaded_texts \
                 = self.data_augmenter.augment_datas(list(datas), list(texts), n_augmentations=abs_n_augs,
                                                     processes=processes, progress_bar=progress_bar)
-
+        
```

Comment:
Something changed here: Remove spaces in line (?)

Reply:
done